src/tools/crossgen2/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1533,9 +1533,18 @@ protected override void AllocAndTransformManagedToNative(ILCodeStream codeStream
         protected override void TransformNativeToManaged(ILCodeStream codeStream)
         {
             ILEmitter emitter = _ilCodeStreams.Emitter;
+
+#if READYTORUN
+            var ansiToString =
+                Context.SystemModule.GetKnownType("System.StubHelpers", "AnsiBSTRMarshaler")
+                .GetKnownMethod("ConvertToManaged", null);
+            LoadNativeValue(codeStream);
```

Comment:
Nit: `LoadNativeValue` and `codeStream.Emit` can be outside the `#if`.

Reply:
Fixed